qtsass/qtsass.py

```diff
@@ -10,12 +13,107 @@
 logger = logging.getLogger(__name__)
 
 
+class Conformer(object):
+    '''Base class for all Text transformations'''
```

Comment:
With regard to all your docstrings, thanks for including such detailed documentation. Per [PEP 257](https://www.python.org/dev/peps/pep-0257/) (the docstring standard), docstrings are required to use double quotes ``"""`` , not single, which is adhered to elsewhere in the codebase. Also per PEP 257, all docstrings should end with a period, and at least according to that (I'm not sure its strictly followed, either here or in the rest of the world) there should be a space both before and after a class docstring, for symmetry.